packages/common/src/safe.external.urls.ts

```diff
@@ -2,14 +2,16 @@ export default {
   playground_help: 'https://github.com/OfficeDev/script-lab/blob/master/README.md',
   ask: 'https://stackoverflow.com/questions/tagged/office-js',
   excel_api:
-    'https://dev.office.com/reference/add-ins/excel/excel-add-ins-reference-overview',
+    'https://docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/excel-add-ins-reference-overview',
```

Comment:
Is there somewhere that we document these URLs?

Reply:
I got them from Alex.  The old ones were no longer functional.